### PR TITLE
RED-28784 documentation for ALTER, MGET and QUERYINDEX

### PR DESCRIPTION
**Pull request by otnielvh**

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -291,3 +342,21 @@ TS.INFO temperature:2:32
 11) rules
 12) (empty list or set)
 ```
+
+### TS.QUERYINDEX
+
+Get all the keys matching the filter list.
+
+```sql
+TS.QUERYINDEX fitler...
```
Comment (K-Jo) [inline docs/commands.md]: Though logical, I missed the `FILTER` in this command.  just a note

File: docs/commands.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 Create a new time-series.
 
 ```sql
-TS.CREATE key [RETENTION retentionSecs] [LABELS field value..]
+TS.CREATE key [RETENTION retentionSecs] [CHUNK_SIZE maxSamplesPerChunk] [LABELS field value..]
```
Comment (K-Jo) [inline docs/commands.md]: We initially thought not to expose chunksize in the docs.  If we specify this, we need a default documentation below (as retenionSecs) and I also wonder if it's not possible to do so in TS.ADD and TS.INCRBY/DECRBY.
Given that we had more PR's on adding this to the docs, I suggest we add it, but than all the way. @gkorland @danni-m

File: docs/commands.md [hunk 3]
```diff
@@ -259,6 +282,34 @@ TS.GET key
 2) "23"
 ```
 
+### TS.MGET
+Multi-get all timeseries matching the specific filter.
```
[the remark this answers is not in the document]
Comment (otnielvh) [reply docs/commands.md]: I agree it is confusing and was trying to hint what MGET stands for. Anyway, your phrasing is much better, so I will use it.

File: docs/commands.md (hunk 2 above)
Comment (otnielvh) [reply docs/commands.md]: Then we should remove `CHUNK_SIZE` from this PR, and after discussing add it in another PR.

File: docs/commands.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ Optional args:
  * retentionSecs - Maximum age for samples compared to current time (in seconds)
     * Default: The global retenion secs configuration of the database (by default, `0`)
     * When set to 0, the series is not trimmed at all
+ * maxSamplesPerChunk - Maximum number of samples contained in each chunk.
```
Comment (K-Jo) [inline docs/commands.md]: this one still needs to go

Comment (otnielvh) [reply docs/commands.md]: Thanks for catching it, it is fixed now

**Pull request merged**